MAUI/Barcode/customization.md

```diff
@@ -0,0 +1,364 @@
+---
+layout: post
+title: Customization in .NET MAUI Barcodes (SfBarcodeGenerator) | Syncfusion
+description: Learn here all about Customization of Syncfusion .NET MAUI Barcodes(SfBarcodeGenerator) control with key features and more.
+platform: maui
+control: SfBarcodeGenerator
+documentation: ug
+---
+
+# Customization in .NET MAUI Barcodes (SfBarcodeGenerator)
+
+## Text customization
+
+### Displaying input value
+
+The provided input value of the barcode can be displayed by enabling its `ShowText` property. By default, the `ShowText` is false.
```

Comment:
The provided input value of the barcode can be displayed by enabling the `ShowText` property.

Reply:
Changes updated.